### PR TITLE
Guide: What happens when a user joins a room at maximum capacity?

### PR DESCRIPTION
**Pull request by CTNicholas**
**TODO**
- [x] Check MAU fact before merging: https://liveblocks.slack.com/archives/C03NG9R49HC/p1691070143353549

### CODE REVIEW

File: docs/pages/platform/limits.mdx [hunk 1]
```diff
@@ -52,5 +59,5 @@ Broadcast event messages have a limit of 1 MB.
 Note that when one real-time data structure is nested inside another, it does
 not count towards the limit. Only the JSON leaves of your data structure count
 towards the limit. For example, if a `LiveList` is nested inside a `LiveObject`,
-the `LiveList` and its contents do not count towards the `LiveObject`'s data
```
Comment (stevenfabre) [inline docs/pages/platform/limits.mdx:55]: Chefkiss

File: guides/pages/what-happens-when-a-user-joins-a-room-at-maximum-capacity.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+first, when is a room at maximum capacity?
+
+- A room is full when it’s hit your maximum simultaneous connections per room
+  limit.
+- Your maximum simultaneous connections limit is defined by your current
+  [plan](/pricing).
+- Any users above that count will not be able to join the room.
+- Any users already in the room will not be affected by another user trying to
+  join.
+- If a user can’t join a room, they will not be counted towards your MAUs.
```
[the remark this answers is not in the document]
Comment (stevenfabre) [reply guides/pages/what-happens-when-a-user-joins-a-room-at-maximum-capacity.mdx:19]: I'm not sure this is through. As long as person connects to Liveblocks, that will be counted.

@GuillaumeSalles can you confirm this please?

**Review by stevenfabre — COMMENTED** [review]
Thank you for putting this together. Lots of people asking for this.

I left a couple of comments here and there. This is good to go, we just need to double check how MAUs get counted when  they're not able to connect to a room that's full.

File: guides/pages/what-happens-when-a-user-joins-a-room-at-maximum-capacity.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (CTNicholas) [reply guides/pages/what-happens-when-a-user-joins-a-room-at-maximum-capacity.mdx:19]: They shouldn't count as an MAU: https://liveblocks.slack.com/archives/C03NG9R49HC/p1691162596334849?thread_ts=1691070143.353549&cid=C03NG9R49HC

**Review by stevenfabre — APPROVED** [review]

**Pull request merged**